docs/i18n.md

```diff
@@ -0,0 +1,91 @@
+# i18n
+
+## Context
+
+Internationalization on Force was introduced as a hackathon project ([PR](https://github.com/artsy/force/pull/9797)).
+
+We are using the [react-i18next](https://react.i18next.com/) internationalization framework for this.
+
+The goal of these change is separating the text content from the codebase, so copy updates can handled in a smoother process.
+
+See [this
+presentation](https://docs.google.com/presentation/d/1VT44uoGAaHX0EcDaYwGHiY8VU-fAkOXK6rSHto/edit#slide=id.gd1b1ff661f_0_12) about the benefits we expect to achieve.
+
+## How to
+
+Locales are defined on [src/System/locales](https://github.com/artsy/force/tree/main/src/System/locales). Currently, the only language supported is en-US.
```

Comment:
@lidimayra - We should move the locals folder into `i18n` so that everything is self contained under one namespace.

Reply:
Makes a lot of sense, thanks @damassi!! ❤️  I pushed this change on [43419bd](https://github.com/artsy/force/pull/10588/commits/43419bdc7aa1e5e2155ad9df9d7b8413f07138b9)